docs/guide/content_rendering/urls_and_routes.md

```diff
@@ -0,0 +1,52 @@
+# URLs and routes
+
+To link to a Location or Content item, use the [`ez_path()`](twig_function_reference/url_twig_functions.md#ez_path) Twig function.
+You need to provide the function with a Location, Content, ContentInfo or [RouteReference](#routereference) object:
+
+``` html+twig
+<p><a href="{{ ez_path(location) }}">Location</a></p>
+
+<p><a href="{{ ez_path(content.contentInfo) }}">Content Info</a></p>
+```
+
+Use [`ez_url()`](twig_function_reference/url_twig_functions.md#ez_url) to get an absolute URL to a Content item or Location:
+
+``` html+twig
+<p><a href="{{ ez_url(location) }}">Location</a></p>
+```
+
+## RouteReference
+
+You can use the [`ez_route()`](twig_function_reference/url_twig_functions.md#ez_route) Twig function
+to create a RouteReference object based on the provided information:
+
+``` html+twig
+{% set routeRef = ez_route("ez_urlalias", { 'locationId': 2 }) %}
```

Comment:
```suggestion
{% set routeReference = ez_route("ez_urlalias", { 'locationId': 2 }) %}
```
The following places either quote or reference the whole name of the object:
https://doc.ibexa.co/en/master/guide/file_management/#download-link-generation
https://doc.ibexa.co/en/master/guide/content_rendering/twig_function_reference/url_twig_functions/#ez_path

Reply:
Here the example creates a custom variable, which could be named anything, but you're right, I changed it for clarity.